package.json

```diff
@@ -40,33 +40,33 @@
   },
   "extensionDependencies": [],
   "dependencies": {
-    "diff": "^4.0.1",
+    "diff": "^4.0.2",
     "json-rpc2": "^1.0.2",
     "moment": "^2.24.0",
-    "semver": "^6.3.0",
+    "semver": "^7.3.2",
     "tree-kill": "^1.2.2",
-    "vscode-debugadapter": "^1.36.0",
-    "vscode-debugprotocol": "^1.36.0",
+    "vscode-debugadapter": "^1.40.0",
+    "vscode-debugprotocol": "^1.40.0",
     "vscode-extension-telemetry": "^0.1.2",
-    "vscode-languageclient": "6.1.0",
+    "vscode-languageclient": "^6.1.3",
```

Comment:
The reason I pinned the version of the language client was that the latest version 6.1.3 because the vscode engine for later versions was specified as 1.43 in vscode-languageclient

Reply:
Ah, I see - makes sense, sorry about that!